components/org.wso2.carbon.identity.conditional.auth.typingdna.functions/src/main/java/org/wso2/carbon/identity/conditional/auth/typingdna/SaveUserInTypingDNAFunctionImpl.java

```diff
@@ -57,6 +58,7 @@ public class SaveUserInTypingDNAFunctionImpl implements SaveUserInTypingDNAFunct
      * @throws TypingDNAAuthenticatorException When unable to retrieve tenant configurations.
      */
     @Override
+    @HostAccess.Export
```

Comment:
what is the usage of this annotation

Reply:
[Graal ](https://github.com/wso2/product-is/issues/20613)